### DISCUSSION TITLE
Add chat/communication layer in github ecosystem

### DISCUSSION THREAD

**Discussion by Developer-piyush** [community/community#20625] [Discussions]
Problem statement: Hard to find contact of developers, sometimes I wanna communicate

I recommend to add direct communication layer(chat system) in GitHub ecosystem that allows everyone to communicate/contact each other just like Instagram/Facebook/twitter chat.. 

For those who don't want to receive message can have privacy option in settings..

**Comment by D7EAD** [comment]
Yes, this has been recommended as a feature addition for a while. Hopefully it gets added soon.

**Comment by ghost** [comment]
With this, the idea of the site is canceled
A place to save codes, work, and group discussions on the public
  Having a private chat is not good... I wish it wasn't there

**Comment by jfullstackdev** [comment]
yup, it's either both sides, now that GitHub Discussion feature is already here, you can use that to build a network of developers that are willing to be part of the group,

imagine if, say, you will be messaging someone without prior contact to that user, he / she might be annoyed by that, 

it's just like this Feedback hub, everyone can communicate and for those who are not interested will not simply visit the repo
